README.md

```diff
@@ -200,6 +200,8 @@ They live in [tests](tests). Run them with grunt: `grunt e2e`.
 
 To build reference documentation into a local folder `jsdoc-docs`: `grunt build-documentation`
 
+To add documentation, use these [tags](https://www.npmjs.com/package/angular-jsdoc#tags-available). See examples [here](https://www.npmjs.com/package/angular-jsdoc#example).
+
```

Comment:
I think we should move this to the dev style guide: https://github.com/medic/medic-docs/blob/master/development/style-guide.md

Maybe also include a link to the jsdocs documentation too.

Reply:
This is done. PR in medic-docs.